packages/ui/src/ui/store/actions/navigation/modals/cell-preview.ts

```diff
@@ -61,13 +31,19 @@ export const showCellPreviewModal = (columnName: string, index: number): ActionT
                         stringLimit: PREVIEW_LIMIT,
                     }),
                 },
-                cancellation: cancelHelper.removeAllAndSave,
+                cancellation: cellPreviewCancelHelper.removeAllAndSave,
             });
 
             const parsed = JSON.parse(json);
 
             const data = parsed.rows[0][columnName];
-            dispatch({type: CELL_PREVIEW.SUCCESS, data: {data}});
+
+            const isIncomplete = data.$incomplete;
+            const incompleteNoticeText = isIncomplete
+                ? 'Unable to load content more than 16MiB. Please use the command bellow to load it locally.'
```

Comment:
It looks like the end of the message will be duplicated by:
[`You could use the command bellow to load it locally`](https://github.com/ytsaurus/ytsaurus-ui/pull/709/commits/fc95d3a354448de118631ccd4f6e1d681fa83a62#diff-75d5eac1a43f5ae20c4ec63e1e668a32e26debc505e43464891c24efcefd8b34R53)